scripts/setup-ports.sh

```diff
@@ -1,3 +1,5 @@
+export securityGroupId="$(aws emr describe-cluster --cluster-id $clusterId | grep sg- | awk '{print $4}')"
```

Comment:
This is excellent, thanks Colin! 

I think $4 is the subnet, but $5 is the first sg. Do you think you could change
```bash
awk '{print $4}'
```
to 
```bash
awk '{print $5}'
```
Also, would you mind removing your hw screenshot from this one (since it's already in the other)?

You rock 🎉